node/chainSimulator/components/processComponents.go

```diff
@@ -42,6 +42,7 @@ type ArgsProcessComponentsHolder struct {
 	NodesCoordinator     nodesCoordinator.NodesCoordinator
 
 	EpochConfig              config.EpochConfig
+	RoundConfig              config.RoundConfig
```

Comment:
can you remove from  file
`../confgs/configs.go` line 122  (`configs.EpochConfig.EnableEpochs.StakingV2EnableEpoch = args.InitialEpoch + 1`)

if the problem is fixed tests should pass without this line

Reply:
done, all tests pass without that line